tsconfig.json

```diff
@@ -6,7 +6,8 @@
         "module": "esnext",
         "moduleResolution": "node",
         "resolveJsonModule": true,
-        "types": ["@testing-library/jest-dom"]
+        "types": ["node", "@testing-library/jest-dom"],
+        "lib": ["es2022", "dom"]
```

Comment:
Not really related to this PR, but it helped to get correct types and syntax highlighting when working on `bin/build_package.ts` file